scripts/generate-charts

```diff
@@ -31,6 +31,14 @@ for f in packages/*; do
 			cp -R ${f}/charts-crd/* charts/$(basename -- ${f})/charts-crd
 		fi
 		helm repo index --merge ./assets/index.yaml --url assets assets
+		
+		# Clean up changes
+		if [[ -f ${f}/package.yaml ]]; then
+			yq w -i ${f}/charts/Chart.yaml 'version' "${version}"
```

Comment:
Whats this doing ?

Reply:
Resetting the version number on the chart by removing the `packageVersion`, e.g. the inverse of https://github.com/rancher/charts/blob/a5baa341654df0e333979f850c2277a95fb8b469/scripts/generate-charts#L11-L15